atlasaction/action.go

```diff
@@ -40,3 +54,63 @@ func MigrateApply(ctx context.Context, client *atlasexec.Client, act *githubacti
 	act.Infof("Run complete: +%v", run)
 	return nil
 }
+
+// MigratePush runs the Github Action for "ariga/atlas-action/migrate/push"
+func MigratePush(ctx context.Context, client *atlasexec.Client, act *githubactions.Action) error {
+	ghContext, err := createContext(act)
+	if err != nil {
+		return fmt.Errorf("failed to read github metadata: %w", err)
+	}
+	buf, err := json.Marshal(ghContext)
+	if err != nil {
+		return fmt.Errorf("failed to create MigratePushParams: %w", err)
+	}
+	params := &atlasexec.MigratePushParams{
+		Name:      act.GetInput("dir-name"),
+		DirURL:    act.GetInput("dir"),
+		DevURL:    act.GetInput("dev-url"),
+		Context:   string(buf),
+		ConfigURL: act.GetInput("config"),
+		Env:       act.GetInput("env"),
+	}
+	_, err = client.MigratePush(context.Background(), params)
+	if err != nil {
+		return fmt.Errorf("failed to push directory: %v", err)
+	}
+	tag := act.GetInput("tag")
+	if tag != "" {
+		params.Tag = tag
+	} else {
+		params.Tag = ghContext.Commit
+	}
+	resp, err := client.MigratePush(context.Background(), params)
+	if err != nil {
+		return fmt.Errorf("failed to push dir tag: %w", err)
+	}
+	act.SetOutput("url", resp)
+	act.Infof("Uploaded migration dir %q to Atlas Cloud\n", params.Name)
```

Comment:
```suggestion
	act.Infof("Uploaded directory %q to Atlas Cloud\n", params.Name)
	act.Infof("To view the directory on Atlas Cloud visit: %s", resp)
```

Is the `\n` here necessary? iirc it can be dropped.

Reply:
Please, paste the Atlas output. Don’t wrap it with extra text

Reply:
@a8m The atlas output is not exposed by the SDK. We only get the URL which is set using the `SetOutput`

so I can either keep this log or completely remove it, I can't print Atlas output

@rotemtam I don't remember why I added the \n. I think it was not added automatically when running as a UT. Not sure about the github action. I'll check both